main.go

```diff
@@ -126,6 +127,11 @@ func main() {
 
 		result, err := c.Collect()
 		if err != nil {
+			fmt.Printf("Error collecting agent metrics, err: %w\n", err)
```

Comment:
Sorry, I missed this
```suggestion
			fmt.Printf("Error collecting agent metrics, err: %s\n", err)
```

`%w` isn't valid for `fmt.Printf`, only `fmt.Errorf`